cmd/install/install.go

```diff
@@ -230,6 +231,7 @@ func NewCommand() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&opts.PullSecretFile, "pull-secret", opts.PullSecretFile, "File path to a pull secret.")
 	cmd.PersistentFlags().StringVar(&opts.ManagedService, "managed-service", opts.ManagedService, "The type of managed service the HyperShift Operator is installed on; this is used to configure different HostedCluster options depending on the managed service. Examples: ARO-HCP, ROSA-HCP")
 	cmd.PersistentFlags().BoolVar(&opts.EnableSizeTagging, "enable-size-tagging", opts.EnableSizeTagging, "If true, HyperShift will tag the HostedCluster with a size label corresponding to the number of worker nodes")
+	cmd.PersistentFlags().BoolVar(&opts.DisableEtcdRecovery, "disable-etcd-recovery", opts.DisableEtcdRecovery, "If true, HyperShift operator will not attempt to recover broken etcd clusters")
```

Comment:
nit: "will not attempt to recover broken etcd clusters" broken clusters makes me thing of cluster where quorum is broken, can we clarify that's not the case and this is only for failing members of working clusters in the desc of the flag?

Reply:
Why is this a install flag? How can I opt out for a single cluster? What about single replica clusters?

Reply:
> How can I opt out for a single cluster to prefer manual intervention? What about single replica clusters?

I can introduce another knob (maybe an annotation). However, I couldn't think of a use case for it.

Reply:
> What about single replica clusters?

We only do the health check/recovery for ha clusters